cli_app/cli_codeable.rb

```diff
@@ -0,0 +1,82 @@
+require 'colorize'
+
+$task_array = []
+$id= 1
+
+def add(choice)
+
+  task_name = choice[2..-1] # Se come el new
```

Comment:
We don't quite understand what you mean by [2..-1]

Reply:
Ok, here we go. When we add a task, we use "t task_example". This format was established by the product owner. We want to catch just "task_example" that's why we "slice/cut/chop" our string and we take since index 2 till end.